index.html

```diff
@@ -20,17 +20,18 @@
   </head>
 
   <body>
-    <div class="header">
+    <header class="header"> 
 
         <a href="#">
           <img
             src="images/bikes-for-refugees_logo.jpg"
             alt="Logo for Bikes for Refugees Scotland"
             class="header__logo"
           />
+          
         </a>
 
-        <div>
+        <nav>
           <ul class="navigation__list">
             <li class="navigation__item">
               <a class="navigation__link" href="#">Home</a>
```

Comment:
A stretch goal - what is an alternative way of styling the **nested** tags? Do we really need to apply an explicit style to `a` here or there is another way of handling this in CSS?